.nuget/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NuGet.CommandLine" version="2.1.0" />

```

Comment:
Why changing this nuget version? We are using it inside our CI setup and upgrading may not work as expected.

Same thing goes for NUnit, etc.